src/test/java/nextstep/subway/line/LineSteps.java

```diff
@@ -3,15 +3,23 @@
 import io.restassured.RestAssured;
 import io.restassured.response.ExtractableResponse;
 import io.restassured.response.Response;
+import nextstep.subway.line.dto.LineRequest;
 import org.apache.groovy.util.Maps;
 import org.springframework.http.MediaType;
 
 import java.util.Map;
 
+import static java.lang.String.valueOf;
+
 public class LineSteps {
 
-    public static ExtractableResponse<Response> 지하철_노선_생성(String name, String color) {
-        Map<String, String> params = Maps.of("name", name, "color", color);
+    public static ExtractableResponse<Response> 지하철_노선_생성(LineRequest lineRequest) {
+        Map<String, String> params = Maps.of(
+                "name", lineRequest.getName()
+                , "color", lineRequest.getColor()
+                , "upStationId", valueOf(lineRequest.getUpStationId())
+                , "downStationId", valueOf(lineRequest.getDownStationId())
+                , "distance", valueOf(lineRequest.getDistance()));
         return RestAssured
                 .given().log().all()
                 .body(params).contentType(MediaType.APPLICATION_JSON_VALUE)
```

Comment:
request dto를 map으로 변환하지 않고 바로 사용하는건 어떨까요?
```suggestion
                .body(lineRequest).contentType(MediaType.APPLICATION_JSON_VALUE)
```